### PR TITLE
поправил дроп

### PR DESCRIPTION
**Pull request by Pasha8914**
# Pull request template

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] branch rebased on `develop`
- [x] styleguide check `npm run validate`
- [x] good naming
- [x] keep simple
- [x] video or screenshot attached
- [x] testing instruction attached
- [x] check your PR once again

## Video or screenshot

###was 
image
image
###become

image
image

### CODE REVIEW

**Comment by sdwas** [comment]
Какие здесь правки?

Screenshot_20190713_144613

File: shared/components/ui/DropDown/DropDown.js [hunk 1]
```diff
@@ -49,51 +43,43 @@ export default class DropDown extends Component {
...
+      infoAboutCurrency: '',
       error: false,
     }
   }
 
-  componentDidMount() {
-    this.showPercentChange1H()
+  componentDidUpdate(prevProps) {
+    const { isToggleActive } = this.props
+    if (prevProps.isToggleActive !== isToggleActive && isToggleActive) this.showPercentChange1H()
```
Comment (sdwas) [inline shared/components/ui/DropDown/DropDown.js]:
```suggestion

    if (prevProps.isToggleActive !== isToggleActive && isToggleActive) {
      this.showPercentChange1H()
    }
```

File: shared/components/ui/DropDown/DropDown.js [hunk 2]
```diff
@@ -49,51 +43,43 @@ export default class DropDown extends Component {
...
   toggle = () => {
     const { isToggleActive, toggleOpen, toggleClose } = this.props
 
-    if (isToggleActive) {
-      toggleClose()
-    }
-    else {
-      toggleOpen()
-    }
+    if (isToggleActive) toggleClose()
```
Comment (sdwas) [inline shared/components/ui/DropDown/DropDown.js]:
```suggestion
    if (isToggleActive) {
      toggleClose()
    } else {
```

File: shared/components/ui/DropDown/DropDown.js [hunk 3]
```diff
@@ -49,51 +43,43 @@ export default class DropDown extends Component {
...
+    else toggleOpen()
```
Comment (sdwas) [inline shared/components/ui/DropDown/DropDown.js]:
```suggestion
      toggleOpen()
    }
```

File: shared/components/ui/DropDown/DropDown.js [hunk 4]
```diff
@@ -106,40 +92,31 @@ export default class DropDown extends Component {
   renderItem = (item) => {
     const { itemRender } = this.props
 
-    if (typeof itemRender === 'function') {
-      return itemRender(item)
-    }
-
+    if (typeof itemRender === 'function') return itemRender(item)
```
Comment (sdwas) [inline shared/components/ui/DropDown/DropDown.js]:
```suggestion
    if (typeof itemRender === 'function') {
      return itemRender(item)
    }
```

File: shared/components/ui/DropDown/DropDown.js [hunk 5]
```diff
@@ -106,40 +92,31 @@ export default class DropDown extends Component {
...
-            items.map(item => { // eslint-disable-line
-              if (item.name === res.symbol) {
-                infoAboutCurrency.push({
-                  name: res.symbol,
-                  change: res.percent_change_1h,
-                })
-              }
+          const itemsName = items.map(el => el.name)
+          result.map(res => {
+            if(itemsName.includes(res.symbol)) infoAboutCurrency.push({
```
Comment (sdwas) [inline shared/components/ui/DropDown/DropDown.js]: Здесь тоже добавь { } к if

File: shared/components/ui/DropDown/DropDown.js [hunk 6]
```diff
@@ -148,15 +125,15 @@ export default class DropDown extends Component {
...
-    })
-
+    const dropDownStyleName = cx('dropDown', { 'active': isToggleActive })
     const linkedValue = Link.all(this, 'inputValue')
 
-    const itemsFiltered = this.props.items
-      .filter(item => item.name.includes(inputValue.toUpperCase()))
-      .filter(item => item.value !== selectedValue)
+    let itemsFiltered = items
+    if(inputValue) {
```
Comment (sdwas) [inline shared/components/ui/DropDown/DropDown.js]:
```suggestion

    if (inputValue) {
```

File: shared/components/ui/DropDown/DropDown.js (hunk 5 above)
Comment (Pasha8914) [reply shared/components/ui/DropDown/DropDown.js]: зачем)

**Pull request merged**